### PR TITLE
Add ChartSeriesAggregationMethod enum

### PR DESCRIPTION
**Pull request by roman-simionov**

### CODE REVIEW

File: js/viz/docs/docchartseries.js [hunk 1]
```diff
@@ -1229,8 +1227,7 @@ var barSeries = {
         /**
         * @name dxChartSeriesTypes_barseries_aggregation_method
         * @publicName method
-        * @type string
-        * @acceptValues 'avg'|'count'|'max'|'min'|'sum'|'custom'
+        * @type ChartSimpleSeriesAggregationMethod
```
Comment (alexanderPolosatov) [inline js/viz/docs/docchartseries.js]: I don't see definition of ChartSimpleSeriesAggregationMethod type.

**Review by alexanderPolosatov — CHANGES_REQUESTED** [review]

File: js/viz/docs/docchartseries.js (hunk 1 above)
Comment (roman-simionov) [reply js/viz/docs/docchartseries.js]: I've added it

**Review by alexanderPolosatov — DISMISSED** [review]

File: js/viz/docs/docchartseries.js [hunk 2]
```diff
@@ -1085,8 +1085,7 @@ var commonSeries = {
         /**
         * @name dxChartSeriesTypes_CommonSeries_aggregation_method
         * @publicName method
-        * @type string
-        * @acceptValues 'avg'|'count'|'max'|'min'|'ohlc'|'range'|'sum'|'custom'
+        * @type ChartCommonSeriesAggregationMethod
```
Comment (San4es) [inline js/viz/docs/docchartseries.js]: Use the "Enums." prefix.
See also @type Enums.DashStyle

File: js/viz/docs/docchartseries.js [hunk 3]
```diff
@@ -1169,8 +1168,7 @@ var areaSeries = {
         /**
         * @name dxChartSeriesTypes_areaseries_aggregation_method
         * @publicName method
-        * @type string
-        * @acceptValues 'avg'|'count'|'max'|'min'|'sum'|'custom'
+        * @type ChartSimpleSeriesAggregationMethod
```
Comment (San4es) [inline js/viz/docs/docchartseries.js]: "Enums.ChartSimpleSeriesAggregationMethod"

File: js/viz/docs/docchartseries.js (hunk 1 above)
Comment (San4es) [reply js/viz/docs/docchartseries.js]: "Enums.ChartSimpleSeriesAggregationMethod"

**Review by San4es — CHANGES_REQUESTED** [review]

File: js/viz/docs/docchartseries.js (hunk 2 above)
Comment (roman-simionov) [reply js/viz/docs/docchartseries.js]: fixed

File: js/viz/docs/docchartseries.js (hunk 3 above)
Comment (roman-simionov) [reply js/viz/docs/docchartseries.js]: fixed

**Review by RomanTsukanov — APPROVED** [review]

**Review by San4es — APPROVED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**